lab_1_classify_by_unigrams/main.py

```diff
@@ -34,7 +43,20 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
-
+    if isinstance(tokens, list):
+        for element in tokens:
+            if isinstance(element, str):
```

Comment:
the same. Check it earlier and swap the conditions, you shoud stop earlier if input is corrupted

Reply:
Done